### DISCUSSION TITLE
cant access folders

### DISCUSSION THREAD

**Discussion by Prathamesh0901** [community/community#102650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i have published my site with github pages.
https://prathamesh0901.github.io/SpotifyClone/
i want to fetch all the songs from Songs folder, but it can't fetch and gives not found error
here is the repo link
https://github.com/Prathamesh0901/SpotifyClone

**Comment by yoannchaudet** [comment]
Hello,

GitHub Pages does not enable directory index listing. Your files are available, but you will need to publish yourself an index.html file to enumerate them or just access them by known URL.

**Reply by Prathamesh0901** [reply]
can you please elaborate
